EasyLearn/config/database.yml

```diff
@@ -4,19 +4,19 @@
 postgresql: &postgresql
   adapter: postgresql
   host: localhost
-  username: username
-  password: password
+  username: p_final
+  password: p_finalBC2021
```

Comment:
This file should not be on GitHub, instead you can use an example file: database.yml.example

Reply:
Done.